components/article/ArticlePublisherMeta.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
+    <meta itemprop="name" content="Koło Naukowe IT Politechniki Krakowskiej" />
+    <div itemprop="logo" itemscope itemtype="http://schema.org/ImageObject">
+      <!-- TODO: add proper logo url and check path -->
+      <meta itemprop="url" content="/temporary/article-author.png" />
```

Comment:
Anyone knows how to include a file from the `static` directory in tag different than `src` or `href`?

Reply:
This is ok for `static` and `~/path/to/file` is for `assets` directory I think. https://nuxtjs.org/guide/assets/